[SEO 14] Fix links to avoid redirects (#225)
KnapsackPro/docs.knapsackpro.com: _posts/2021-03-19-auto-scaling-buildkite-ci-build-agents-for-rspec-run-parallel-jobs-in-minutes-instead-of-hours.md

    parallelism: 2
{% endhighlight %}

Please note that you should hide your credentials like the Knapsack Pro API token and not commit it into your repository. You can refer to the [Buildkite secrets documentation](https://buildkite.com/docs/pipelines/security/secrets/managing).

## An advanced Buildkite config with Elastic CI Stack for AWS

> When you want to run your big RSpec project on dozen or even hundreds of parallel machines you need powerful resources. In such a case, you can follow the [Buildkite tutorial about AWS setup](https://buildkite.com/docs/agent/v3/elastic-ci-aws). The Elastic CI Stack for AWS gives you a private, autoscaling Buildkite Agent cluster in your own AWS account.

Reviewer: ```suggestion
If you want to run your RSpec project on dozen or even hundreds of parallel machines, you need powerful resources. In such a case, you can follow the [Buildkite tutorial about AWS setup](https://buildkite.com/docs/agent/v3/elastic-ci-aws). The Elastic CI Stack for AWS gives you a private, autoscaling Buildkite Agent cluster in your own AWS account.
```
